### DISCUSSION TITLE
This crawler does it only work on laravel?

### DISCUSSION THREAD

**Discussion by thiagovieiradev** [spatie/crawler#433] [General]
Does it work in other PHP frameworks or only in laravel?

I tried to install by composer but this error appeared:
`Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - spatie/crawler[6.0.0, ..., 6.0.2] require illuminate/collections ^8.17 -> satisfiable by illuminate/collections[v8.17.0, ..., v8.83.25].
    - illuminate/collections[v8.17.0, ..., v8.83.25] require illuminate/contracts ^8.0 -> satisfiable by illuminate/contracts[v8.0.0, ..., v8.83.25].
    - illuminate/contracts[v8.0.0, ..., v8.83.25] require psr/container ^1.0 -> found psr/container[1.0.0, 1.1.0, 1.1.1, 1.1.2] but the package is fixed to 2.0.2 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires spatie/crawler ^6.0 -> satisfiable by spatie/crawler[6.0.0, 6.0.1, 6.0.2].`

What i need to do? I'm using codeigniter framework.

**Comment by freekmurze** [comment]
This package is not framework specific.

**Reply by thiagovieiradev** [reply]
Hello, thanks. I update my answer.

**Comment by ao-jhelmich** [comment]
What you need to do is install the dependecies, those illuminate packages are not laravel specific
